src/packages/core/property-editor/uis/tree-picker/config/start-node/property-editor-ui-tree-picker-start-node.element.ts

```diff
@@ -1,20 +1,54 @@
-import { html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
-import { UmbTextStyles } from "@umbraco-cms/backoffice/style";
+import { StartNode, UmbInputStartNodeElement } from '@umbraco-cms/backoffice/components';
+import { html, customElement, property, state } from '@umbraco-cms/backoffice/external/lit';
+import { UmbPropertyEditorConfigCollection } from '@umbraco-cms/backoffice/property-editor';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
 import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
 
 /**
  * @element umb-property-editor-ui-tree-picker-start-node
  */
 @customElement('umb-property-editor-ui-tree-picker-start-node')
 export class UmbPropertyEditorUITreePickerStartNodeElement extends UmbLitElement {
-	@property()
-	value = '';
+	@property({ type: Object })
+	value: StartNode = {
+		type: 'content',
+	};
 
-	@property({ type: Array, attribute: false })
-	public config = [];
+	@state()
+	private _options: Array<Option> = [];
+
+	@property({ attribute: false })
+	public set config(config: UmbPropertyEditorConfigCollection | undefined) {
+		if (!config) return;
+
+		const listData = config.getValueByAlias('items') as Array<Option>;
+		if (!listData.length) return;
+
+		this._options = listData.map((item): Option => {
+			const option: Option = { value: item.value, name: item.name, selected: false };
+			if (item.value === this.value.type) {
+				option.selected = true;
+			}
+			return option;
+		});
+	}
+
+	#onChange(event: CustomEvent) {
+		const target = event.target as UmbInputStartNodeElement;
+
+		this.value = {
+			type: target.startNodeType,
+			id: target.startNodeId,
+		};
+
+		this.dispatchEvent(new CustomEvent('property-value-change'));
```

Comment:
If you feel like it, you could create a new custom event, like we have for the change event, and use it here instead of the hardcoded value. I think there might be other places where we also hardcode the event type, so it could also be in a new PR too.

Reply:
Sure, i will do it after this PR gets completed, also a PR for dynamic paths.